bin/hermit.hcl

```diff
@@ -1,4 +1,4 @@
 env = {
-  "CLOUDBEAT_VERSION": "9.0.0",
+  "CLOUDBEAT_VERSION": "9.1.0",
```

Comment:
This will be backported to 8.x?

Reply:
No, I'll remove this change, and the version bump will be done in a [separate PR](https://github.com/elastic/cloudbeat/pull/3088).